### DISCUSSION TITLE
tutorials/navigation/navigation_using_navigationlinks

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#226] [User-contributed notes]
# tutorials/navigation/navigation_using_navigationlinks

NavigationLinks are used to connect navigation mesh polygons from NavigationRegion2D and NavigationRegion3D over arbitrary distances for pathfinding. NavigationLinks are also used to consider movem...

https://docs.godotengine.org/en/stable/tutorials/navigation/navigation_using_navigationlinks.html

**Comment by liamhendricks** [comment]
Hopefully this helps others, but I have noticed that the `start_position` and `end_position` need to be very close to the y-axis of the respective nav regions in order for this to work. Like, way closer than the `navigation/2d_or_3d/default_link_connection_radius` seems to indicate. Not really sure if this is a bug or not, but just be aware.

**Comment by VojtaStruhar** [comment]
Also I noticed if the start and end positions of a link are in the same region, it seems to get discarded. If you have overlapping regions (like going up and down stairs in 2D), be wary of this.

**Comment by Wilker-uwu** [comment]
what practice is best when dealing with intended links which may have multiple segments and/or be on closed ring paths?

**Comment by BaconEggsRL** [comment]
"A navigation link does not provide any specialized movement through the link. Instead, when an agent reaches the position of a link, game code needs to react (e.g. through area triggers) and provide means for the agent to move through the link"

Some examples of how to use this would be helpful

**Comment by RonYanDaik** [comment]
>    `2D and 3D versions of NavigationJumplinks nodes are available as NavigationLink2D and NavigationLink3D respectively`

What is `NavigationJumplink`?
